recipe/meta.yaml

```diff
@@ -13,27 +13,42 @@ source:
   sha256: 7697b1d64cd9b8eb5abd9e7594e318a80f1382f45d879c425944175697b29db2
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - cookiecutter = cookiecutter.cli:main
+    # master has changed it's setup a bit as of 4a64e48... will have to fix
+    # - cookiecutter = cookiecutter.__main__:main
 
 requirements:
   build:
     - python
+    - setuptools
   run:
-    - future >=0.15.2
     - binaryornot >=0.2.0
-    - jinja2 >=2.7
     - click >=5.0
-    - whichcraft >=0.3.1
-    - poyo >=0.1.0
+    - future >=0.15.2
+    - jinja2 >=2.7
     - jinja2-time >=0.1.0
+    - poyo >=0.1.0
+    - python
+    - whichcraft >=0.3.1
 
 test:
-  files:
-    - requirements.txt
   requires:
-    - pytest-cov
+    - git
     - pytest
+    - pytest-cov
+    - pytest-mock
+    - python
+    - freezegun
+  commands:
+    # Run basic commands.
+    - cookiecutter --version
+    - cookiecutter --help
+    # Run test suite.
+    - cd {{ environ["SRC_DIR"] }}
+    - py.test --cov=cookiecutter -k "not _hg_ and not mercurial and not test_make_sure_path_exists and not pypackage_git"
```

Comment:
I guess we could actually test with mercurial on py27... not right now, though :]